karma.conf.js

```diff
@@ -0,0 +1,57 @@
+module.exports = function (config) {
+  var path = require('path')
+  var node_modules_dir = path.join(__dirname, 'node_modules')
+  var deps = [
+    'deps/forge.bundle.js'
```

Comment:
Could you create an issue saying that this can be replaced once issues X Y and Z get solved on node-forge? Thank you :)